src/bidmanager.js

```diff
@@ -43,6 +43,7 @@ function getBidders(bid) {
 function bidsBackAdUnit(adUnitCode) {
   const requested = $$PREBID_GLOBAL$$._bidsRequested
     .map(request => request.bids
+      .filter(adUnitsFilter.bind(this, $$PREBID_GLOBAL$$._adUnitCodes))
```

Comment:
Actually if we use `.bind` for `processCallbacks` to support a single ad unit code or all ad unit codes (for `bidsBackAdUnit` and `bidsBackAll` respectively) then we should use the same partial application pattern in all of these cases. This is probably preferable as the ad unit codes  to use as a filter can be set as instance data on the `requestBids` function and we can remove the `_adUnitCodes` global property.